src/util/util.js

```diff
@@ -87,6 +103,28 @@ export function getBounds(points: Point[]): { min: Point, max: Point} {
     };
 }
 
+/**
+ * Returns the 2D square distance between an AABB defined by min and max and a point.
+ * If point is null or undefined, the AABB distance from the origin (0,0) is returned.
+ *
+ * @param {Point} min The minimum extent of the AABB.
+ * @param {Point} max The maximum extent of the AABB.
+ * @param {Point} point The point to compute the distance from, may be undefined.
```

Comment:
I think brackets is jsdoc syntax for optional? https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

```suggestion
 * @param {Point} [point] The point to compute the distance from, may be undefined.
```